docs/topics/sealed-classes.md

```diff
@@ -1,37 +1,18 @@
 [//]: # (title: Sealed classes)
 
-_Sealed_ classes represent restricted class hierarchies that provide more control over inheritance.
+_Sealed_ classes and interfaces represent restricted class hierarchies that provide more control over inheritance.
 All subclasses of a sealed class are known at compile time. No other subclasses may appear after
 a module with the sealed class is compiled. For example, third-party clients can't extend your sealed class in their code.
 Thus, each instance of a sealed class has a type from a limited set that is known when this class is compiled.
 
-In some sense, sealed classes are similar to [`enum` classes](enum-classes.md): the set of values
+The same works for sealed interfaces and their implementations: once a module with a sealed interface is compiled,
+no new implementations can appear.
+
+In some sense, sealed classes are similar to [`enum`](enum-classes.md) classes: the set of values
 for an enum type is also restricted, but each enum constant exists only as a _single instance_, whereas a subclass
 of a sealed class can have _multiple_ instances, each with its own state.
 
-To declare a sealed class, put the `sealed` modifier before its name.
-
-```kotlin
-sealed class Expr
-data class Const(val number: Double) : Expr()
-data class Sum(val e1: Expr, val e2: Expr) : Expr()
-object NotANumber : Expr()
-```
-
-A sealed class is [abstract](classes.md#abstract-classes) by itself, it cannot be instantiated directly and can have `abstract` members.
-
-Sealed classes are not allowed to have non-`private` constructors (their constructors are `private` by default).
-
-## Sealed interfaces
-
-> Sealed interfaces are [Experimental](components-stability.md). They may be dropped or changed at any time.
-> Opt-in is required (see the details [below](#try-sealed-interfaces-and-package-wide-hierarchies-of-sealed-classes)), and you should use them only for evaluation purposes.  We would appreciate your feedback on them in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42433).
->
-{type="warning"}
-
-Interfaces can be declared `sealed` as well as classes. The `sealed` modifier works on interfaces the same way:
-all implementations of a sealed interface are known at compile time. Once a module with a sealed interface is compiled,
-no new implementations can appear.
+To declare a sealed class or interface, put the `sealed` modifier before its name.
```

Comment:
It seems that we need to agree on whether we should put a colon in such cases or not :)

Reply:
Good idea, let's state this in our guidelines. 
I've checked docs of some popular languages (Java, Python, C#). They all use a colon between such explanations and examples.